engine/src/cmd/damageable.cpp

```diff
@@ -27,15 +27,23 @@
 
 #include "gfx/vec.h"
 #include "vs_math.h"
-#include "xml_support.h"
+#include "game_config.h"
 #include "vs_globals.h"
 #include "configxml.h"
 #include "unit_armorshield.h"
 #include "gfx/vec.h"
-#include "unit_damage.h"
+#include "lin_time.h"
 
 #include <algorithm>
 
+// TODO: make GameConfig support sub sections
```

Comment:
Please make sure there is an issue filed, and then put a link to the issue in the code too.

Reply:
See #358. Done